### ISSUE TITLE
Build fails (r_types.h: No such file or directory)

### ISSUE THREAD

**Issue by GustavoLCR** [radare/radare2#11897]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | WSL Debian 9 x86/64

### Expected behavior
   Build finishes successfully
### Actual behavior
   Build fails as it cannot find r_types.h

![screen](https://user-images.githubusercontent.com/23562137/47276184-c067d200-d58b-11e8-9b3e-e7ac865089a4.png)
### Steps to reproduce the behavior 
   Run the sys/install.sh

### Console log
[console_log.zip](https://github.com/radare/radare2/files/2500038/console_log.zip)

**Comment by radare** [comment]
there's no need to run this script as root. in fact its totally unrecommended.

but i dont see why it should fail, unless you have a broken checkout.

try with git clean -xdf; git checkout; git pull

is this showing any error?

**Comment by GustavoLCR** [comment]
I tried running the commands, no errors, then running install.sh and the problem continued. 
I looked into `/libr/include` and it seemed like a bunch of files were still missing if compared with the github repository, so I ran 
`git fetch --all; git reset --hard origin/master; git pull origin master`
then the problem finally was solved and it managed to install. I really don't know why or how those files were deleted in the first place, as I didn't touch the folder in 2+ months after one install.
